### PR TITLE
Replaced Calendar with Past Code Clubs Section

### LINKED ISSUE #25: Add Past Clubs Timeline
Include pictures of what the girls did, links to projects, and short description of what the girls learned

### PR DESCRIPTION
**Pull request by troynguyen8**
Added basic past code clubs section. Will work on less simple design in the future.

Closes #25

### CODE REVIEW

**Comment by troynguyen8** [comment]
Aakriti mentioned she would prefer the more recent clubs on top. Will make sure to make this change before merge, but feel free to review

**Comment by justinjstark** [comment]
Does anybody use our calendar? I try to keep it up to date with out clubs and events going on. I can see value in keeping it. But maybe there's a better way to radiate information about current meetings and events!?

**Comment by troynguyen8** [comment]
@justinjstark I don't want to completely get rid of the calendar in the long run, but I was thinking that the iframe looks really out of place right now and eventually we can maybe make some requests and style as we desire

File: src/index.html [hunk 1]
```diff
@@ -63,20 +63,64 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
 
-      <section id="calendar" class="bg-pink col-12">
-        <iframe src="https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=400&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=eh2akbgitkihie7k4svqg256u8%40group.calendar.google.com&amp;color=%235229A3&amp;ctz=America%2FChicago"
-          style="border-width:0" width="100%" height="400" frameborder="0" scrolling="no"></iframe>
+      <section id="past-clubs" class="bg-pink col-12 py-5">
+        <h2 class="display3 text-center mb-5">Our Past Code Clubs</h2>
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-6 col-xs-8">
+            <ul class="timeline">
+              <li>
```
Comment (justinjstark) [inline src/index.html]: Let's order the timeline reverse-chronologically so the recent clubs are at the top

Comment (justinjstark) [inline src/index.html]: How about adding current clubs to the list so it's clear our organization is still active?

**Comment by troynguyen8** [comment]
Closing PR. Addressed by Justin's branch, #37

**Pull request closed without merging**